src/index.html

```diff
@@ -1,16 +1,50 @@
-<!doctype html>
-<html lang="en">
-  <head>
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head class="page__head">
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.scss">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+    <main class="main">
+      <form
+        class="search__form search--form--big"
+        data-qa="big"
+        action="#"
+        method="post"
+      >
+        <label class="search--form--label big__form--label">
+          <input
+            type="text"
+            class="input--big search--form__input"
+            name="field1"
```

Comment:
Please fix previous comment. Fix all cases
```suggestion
            name="big-input"
```
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/ee19248d-339f-47b6-a2ac-24959e6566a6)